.github/workflows/publishVSCode.yml

```diff
@@ -1,26 +1,27 @@
 name: Publish in Microsoft Marketplace
 on:
   release:
-    types: [published, edited]
+    # This limits the workflow to releases that are not pre-releases 
+    # From the docs: A release was published, or a pre-release was changed to a release.
+    types: [ released ]
```

Comment:
doc says it but Randi found info to make this just run for real releases so I could remove the pre-release filtering.  Also removed the edited that didn't ever fire the workflow as expected anyway.